scripts/requirements-ci.sh

```diff
@@ -29,3 +29,7 @@ pip3 freeze > $OUT_FILE
 
 deactivate
 rmvirtualenv pip-freeze-ci-venv
+
+
+# Sphinx==2.4.4
+# breathe==4.14.2
```

Comment:
Just curious.
Why add those lines as they are anyway commented out ?
Also, the syntax used here seems wrong for a bash script.

Reply:
I can remove them.
what's wrong with the syntax?

Reply:
That doing `breathe==4.14.2` in bash would not work (as if you removed the `#`), it seems to be the syntax you would use in a pip requirements.txt file 😉